config.txt

```diff
@@ -63,6 +63,6 @@ dtoverlay=dwc2
 # information across reboots in DRAM
 dtoverlay=ramoops
 
-# Enable the UART (/dev/ttyS0)
+# Enable the UART (/dev/ttyAMA0)
 enable_uart=1
```

Comment:
Is it necessary to leave these two lines?

Reply:
This sounds convenient to have since it seems more likely to use those pins as a UART rather than a GPIO.